VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.edx.mobile.discussion;
+
+import android.app.Activity;
+import android.support.v4.widget.TextViewCompat;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.edx.mobile.R;
+import org.edx.mobile.third_party.iconify.IconDrawable;
+import org.edx.mobile.third_party.iconify.Iconify;
+
+public abstract class DiscussionUtils {
+    /**
+     * Sets the state, text and icon of the new item creation button on discussion screens
+     *
+     * @param activity  The current activity
```

Comment:
In fact the `Context` can be retrieved from the `TextView` via the `getContext()`, so this parameter can just be removed I guess.